### ISSUE TITLE
[whitespace] bug in template strings with = operator

### ISSUE THREAD

**Issue by adidahiya** [palantir/tslint#313]
The following code throws a tslint "missing whitespace" error:

```
var str = `
<div class="foo"></div>
`;
```

It accepts the code if you add a space after `class=`

```
var str = `
<div class= "foo"></div>
`;
```

This should not be an error.

**Comment by gscshoyru** [comment]
This should be fixed by #311 .

**Comment by adidahiya** [comment]
cool, thanks
